docker/BUILD

```diff
@@ -5,6 +5,17 @@ load("//docker:debug_docker.bzl", "debug_docker_build")
 # Use "manual" target tag to skip rules in the wildcard expansion
 
 pkg_tar(
+    name = "agent_tar",
+    extension = "tar.gz",
+    files = [
+        "//cmd/agent",
```

Comment:
Ok, Bazel doesn't allow renaming binaries. Is this really important to assign a long name?